text/0000-local-loop-bindings.md

```diff
@@ -0,0 +1,218 @@
+- Feature Name: local_loop_bindings
+- Start Date: 2018-12-25
+- RFC PR: (leave this empty)
+- Rust Issue: (leave this empty)
+
+# Summary
+[summary]: #summary
+
+This will allow an extended syntax for `loop` to accept local variables, which change for each iteration to simplify more complicated loop constructs and avoid mutable state.
+
+
+# Motivation
+[motivation]: #motivation
+
+The new syntax is inspired by `loop` in the upcoming release of the [scopes programming language](scopes.rocks).
```

Comment:
It will basically the same in the next version, and the syntax may be confusing, so I thought, I should leave it out. But I'll also add an example, when you think so.